pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/user/UserDao.java

```diff
@@ -110,6 +112,26 @@ public int create(UserDto userDto) {
         });
     }
 
+    public static void update(int userId, UserDto userDto) {
+        String email = userDto.getEmailOrThrow();
+        String errorMsg = "Error updating user " + email;
+        int res = inTransaction(conn -> {
+            try (PreparedStatement stmt = conn.prepareStatement(SQL_UPDATE_USER)) {
+                stmt.setString(1, userDto.getName().orElse(null));
+                stmt.setString(2, userDto.getPhoneNumber().orElse(null));
```

Comment:
Since its updating existing user, aren't we nullifying the existing phone number if NOT passed ?
May be it will handle scenario where we want to remove existing phone number . 
Pointing out just in case

Reply:
See my answer above. Open to whatever makes sense based on our conventions.